alica_engine/include/engine/teammanager/Agent.h

```diff
@@ -28,36 +28,41 @@ class Agent {
     friend ::alica::TeamObserver;
```

Comment:
@gauthamm the prefixed :: denotes the root namespace to avoid any possible namespace collisions. Kind of like giving an absolute path instead of a relative path. More safe and less prone to error in cases of there being another namespace of the same name, say, in another library.